Dockerfile

```diff
@@ -19,41 +20,112 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
-# Set this to the version ORT should report.
-ARG ORT_VERSION="DOCKER-SNAPSHOT"
+#------------------------------------------------------------------------
+# Use OpenJDK Eclipe Temurin Ubuntu LTS
+FROM eclipse-temurin:11-jdk AS build
```

Comment:
We've already used `eclipse-temurin:11-jdk-jammy` before, so "Base main Docker on OpenJDK 11 Eclipse Temurin" in the commit message is misleading.

Reply:
No longer applicable as commit was squashed.